### PR TITLE
Optimize and add <title> to SVGs

### LINKED ISSUE #3987: Improvement: Add hover title for thumbs-down icon (means "nonstandard")
(Example) MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Force_Touch_events

#### What information was incorrect, unhelpful, or incomplete?
Here, and anywhere it's used, it would be best if the thumbs-down icon next to a given feature had a hover title, such as "Nonstandard". (I infer it means "nonstandard" based on the SVG's `id="icon-nonstandard"`)

#### Specific section or headline?
N/A, this applies site-wide.

#### What did you expect to see?
I expected, when hovering my mouse over the thumbs-down icon, to see a hover title explaining what it meant.

#### Did you test this? If so, how?
Tested by hovering the icon, and inspecting the HTML.

#### Suggested fix
It looks like the HTML `title` attribute doesn't work on SVGs, but we could add a `<title>` to inside the SVG.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/force_touch_events`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Force_Touch_events
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/force_touch_events/index.html
* Last commit: https://github.com/mdn/content/commit/cbe151a06d6e5b4d1fbb46081bd16e69ef4c1630
* Document last modified: 2020-09-15T15:48:51.000Z

</details>

### PR DESCRIPTION
**Pull request by KartikSoneji**
Optimizes SVGs by removing unused tags.
Adds `<title>` for accessibility.

I didn't do the SVGs in `/client/src/plus` since MDN plus is not released yet.

Closes #3987

### CODE REVIEW

File: client/public/assets/badges.svg [hunk 1]
```diff
@@ -1,13 +1,25 @@
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-    <symbol id="icon-experimental" viewBox="0 0 100 100">
-        <path d="M90.72 82.34c4.4 7 1.29 12.66-7 12.66H16.25C8 95 4.88 89.31 9.28 82.34l29.47-46.46V12.5H35A3.75 3.75 0 0135 5h30a3.75 3.75 0 010 7.5h-3.75v23.38zM45.08 39.86L29.14 65h41.72L54.92 39.86l-1.17-1.81V12.5h-7.5v25.55z" fill="currentColor"/>
-    </symbol>
+<svg xmlns="http://www.w3.org/2000/svg">
+  <symbol id="icon-experimental" viewBox="0 0 100 100">
+    <title>Experimental</title>
```
Comment (Ryuno-Ki) [inline client/public/assets/badges.svg:3]: As far as I know, `<title>` needs to be the first child of `<svg>`.

File: client/src/assets/clippy.svg [hunk 2]
```diff
@@ -1 +1,11 @@
-<svg viewBox="0 0 1000 1000" height="30" width="30" xmlns="http://www.w3.org/2000/svg"><path d="M704 896H64V320h640v192h64V192c0-35-29-64-64-64H512C512 57 455 0 384 0S256 57 256 128H64c-35 0-64 29-64 64v704c0 35 29 64 64 64h640c35 0 64-29 64-64V768h-64v128zM192 192h64s64-29 64-64 29-64 64-64 64 29 64 64 32 64 64 64h64s64 29 64 64H128c0-39 28-64 64-64zm-64 512h128v-64H128v64zm448-128V448L320 640l256 192V704h320V576H576zM128 832h192v-64H128v64zm320-448H128v64h320v-64zM256 512H128v64h128v-64z"/></svg>
\ No newline at end of file
+<svg
+  viewBox="0 0 1000 1000"
+  height="30"
+  width="30"
+  xmlns="http://www.w3.org/2000/svg"
```
Comment (Ryuno-Ki) [inline client/src/assets/clippy.svg:5]: You might want to add `role="img"` and/or link the `<title>` with `aria-labelledby`.

**Review by Ryuno-Ki — COMMENTED** [review]
I'd like to have a deployed version somewhere so I can check with a screenreader.

File: client/src/assets/clippy.svg (hunk 2 above)
Comment (KartikSoneji) [reply client/src/assets/clippy.svg:5]: I have added the `role="img"` attributes.
Is `aria-labelledby` needed even with the `<title>` tag?

Comment (Ryuno-Ki) [reply client/src/assets/clippy.svg:5]: I'll give it a try and let you know.

Comment (Ryuno-Ki) [reply client/src/assets/clippy.svg:5]: Okay, the `<title>` was read everywhere except the badges (which is to be expected, given that it is a `<symbol>` as parent).

No need for aria-labelledby!

**Review by Ryuno-Ki — APPROVED** [review]
I tested with TalkBack on Android and a local checkout using `git fetch upstream pull/4034/head:optimize-svgs && git checkout optimize-svgs` (mdn/yari is „upstream” in my setup).

File: client/src/assets/clippy.svg (hunk 2 above)
Comment (KartikSoneji) [reply client/src/assets/clippy.svg:5]: Chrome does recognize `<title>` in `<symbol>`.

image

**Comment by peterbe** [comment]
@schalkneethling ping r?

**Review by schalkneethling — APPROVED** [review]
🎉  Thanks so much!

**Pull request merged**